mmcv/ops/csrc/common/cuda/feature_refine_cuda_kernel.cuh

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) OpenMMLab. All rights reserved.
+// Modified from
+// https://github.com/SJTU-Thinklab-Det/r3det-on-mmdetection/blob/master/mmdet/ops/fr/src/feature_refine_kernel.cu
+#ifndef FEATURE_REFINE_CUDA_KERNEL_CUH
+#define FEATURE_REFINE_CUDA_KERNEL_CUH
+
+#ifdef MMCV_USE_PARROTS
+#include "parrots_cuda_helper.hpp"
+#else
+#include "pytorch_cuda_helper.hpp"
+#endif
+
+template <typename scalar_t>
+__global__ void feature_refine_forward_kernel(
```

Comment:
```suggestion
__global__ void rotated_feature_align_forward_kernel(
```

Reply:
Should we need to replace all the `feature_refine` with `rotated_feature_align`?

Reply:
Note that we also need to rename the filename.